test/test_gpu_tensorcore.cpp

```diff
@@ -2350,141 +2350,143 @@ TEST_F(NVFuserTest, FusionAmpereMatmulSmemEpilogue_CUDA) {
   constexpr bool ignore_occupancy_drop = true;
   // Keep multiples of 8 to keep vectorizable.
   int M = 4096, N = 4096, K = 4096;
-  for (auto layout : kAllSupportedMmaLayout) {
-    Fusion fusion;
-    FusionGuard fg(&fusion);
+  for (int num_stages : {0, 2}) {
```

Comment:
Could you add a comment here and in the definition of `MatmulParams::DoubleBufferOptions` about what `num_stages <= 1` means?

Reply:
Added comments.